### ISSUE TITLE
Suggestion: Option to delete and recreate the symlink when reinstalling SteamCMD

### ISSUE THREAD

**Issue by hk21702** [RimSort/RimSort#575]
When a user is re-installing SteamCMD for whatever reason, and a symlink is detected, they should be given the option to delete and re-create it. 

Additionally, just in general, there should be a better description of what a symlink is and why you should create it if you intend to use steamCMD to download files.

**Comment by Patryk-Malinowski** [comment]
I have this implemented and I know what a symlink is, but I am unaware what the benefits of creating that symlink are when it comes to using steamCMD to download files...

Overall I am quite new to RimSort (and steamCMD) so I may be missing something obvious.

Would you be able to give me a brief explanation please? (For myself and so that I can hopefully come up with an appropriate explanation as to why the user should create it)

**Comment by hk21702** [comment]
The symlink goes to the local mods folder so that steamcmd downloads and installs the mods there.

Though thinking about it, I'm not entirely convinced that it's necessary because I think you can just tell steamcmd where to install via an arg... I'd need to ask the others as to why the same thing can't just be achieved that way.

**Comment by hk21702** [comment]
Just confirmed it. The symlink/junction is required. The force install dir only has to do with prefixes, not the actual install location for workshop downloads.
